packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -155,6 +229,12 @@ class Navigator extends StatefulComponent {
     });
   }
 
+  /// Calls callback immediately to create a navigator transaction.
```

Comment:
Can you add the following or something like it to each static in this class other than the openTransaction one?:

```
///
/// Uses [openTransaction()]. Only one transaction will be executed per frame.
```